module/VuFind/src/VuFind/Search/Base/Results.php

```diff
@@ -614,8 +623,6 @@ public function getExtraData(): array
      * @param array $data Extra data
      *
      * @return void
-     *
-     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
```

Comment:
I had no idea why this suppresswarning was present so I removed it.

Reply:
Maybe the annotation was added before the logic to log empty data was added, and then we forgot to remove it. In any case, you're right that it's not needed. Thanks!